libminifi/src/agent/agent_docs.cpp

```diff
@@ -0,0 +1,32 @@
+/*** Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
very minor, but there should be a newline after the second asterisk

Reply:
added newline